addons/wp/functions/fnc_taskReset.sqf

```diff
@@ -72,21 +70,37 @@ _group setVariable [QGVAR(taskAssaultMembers), nil, true];
     _x setVariable [QEGVAR(danger,disableAI), nil, true];
     _x setVariable [QEGVAR(danger,forceMove), nil, true];
 
-    // rejoin group
-    [_x] joinSilent _groupNew;
-    _x doFollow (leader _x);
+    // rejoin
+    _x doFollow _leader;
 } count _units;
 
+// exit on soft reset
+if (_softReset) exitWith {
+    _units joinSilent _group;
+    _group
+};
+
+// make new group + adopt name
+private _groupNew = createGroup [side _group, true];
+_groupNew setGroupIdGlobal [groupId _group];
+_groupNew setFormation (formation _group);
+
+// rejoin group
+_units joinSilent _groupNew;
+
 // reset lambs variable
 _groupNew setVariable [QEGVAR(danger,disableGroupAI), nil];
-_groupNew setVariable [QEGVAR(danger,enableGroupReinforce), _reinforce, true];
+_groupNew setVariable [QEGVAR(danger,enableGroupReinforce), _group getVariable [QEGVAR(danger,enableGroupReinforce), false], true];
```

Comment:
shouldn't we also reset those on a soft reset?

Reply:
also what i just noticed enableGroupReinforce should be a main variable and not a danger one if we use it in WP

Reply:
1. I will fix.
2. Lets talk about reinforcement logic before Sunday sometime :)